mmv1/third_party/terraform/services/datastream/data_source_google_datastream_private_connection_test.go

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
```

Comment:
The copyright comments are not needed and will be added automatically when generating the provider.